rollup.config.js

```diff
@@ -0,0 +1,40 @@
+import { terser } from 'rollup-plugin-terser'
+import babel from '@rollup/plugin-babel'
+
+export default [
+  getConfigFor('lazy-noscript'),
+  getConfigFor('lazy-polyfill')
+]
+
+/**
+ * @param {string} file
+ * @returns {import('rollup').RollupOptions}
+ */
+function getConfigFor (file) {
+  return {
+    input: `./dist/runtime/scripts/${file}.js`,
+    output: [
+      {
+        file: `./dist/runtime/scripts/${file}.min.js`,
+        format: 'iife',
+        plugins: [
+          terser()
+        ]
+      }
+    ],
+    plugins: [
+      babel({
```

Comment:
Off-topic: But esbuild itself has the ability for transpiling 🙈

Reply:
Yes, now that we're targeting safari 12 the output is identical ... 👍